variables.tf

```diff
@@ -4,30 +4,55 @@
 # You must provide a value for each of these parameters.
 # ------------------------------------------------------------------------------
 
-variable "subnet_id" {
-  type        = string
-  description = "The ID of the AWS subnet to deploy into (e.g. subnet-0123456789abcdef0)."
+variable "users" {
+  description = "A map whose keys are the usernames of each non-admin user and whose values are a map containing supported user attributes.  The only currently-supported attribute is \"require_mfa\" (boolean).  Example: { \"firstname1.lastname1\" = { \"require_mfa\" = false }, \"firstname2.lastname2\" = { \"require_mfa\" = true }, \"firstname3.lastname3\" = { \"require_mfa\" = false } }"
+  type        = map(map(string))
 }
 
 # ------------------------------------------------------------------------------
 # OPTIONAL PARAMETERS
 #
 # These parameters have reasonable defaults.
 # ------------------------------------------------------------------------------
-variable "ami_owner_account_id" {
+
+variable "aws_region" {
+  default     = "us-east-1"
+  description = "The AWS region where the non-global resources are to be provisioned (e.g. \"us-east-1\")."
   type        = string
-  description = "The ID of the AWS account that owns the Example AMI, or \"self\" if the AMI is owned by the same account as the provisioner."
-  default     = "self"
 }
 
-variable "aws_availability_zone" {
+variable "non_self_admin_users" {
+  default     = []
+  description = "A list containing the usernames of non-admin users that are not allowed to administer their own accounts.  Example: [ \"service-account1\", \"service-account2\", \"service-account3\" ]"
+  type        = list(string)
+}
```

Comment:
Any thoughts on adding this capability as another attribute in the `object` associated with each `username` in `var.users`?  That seems cleaner and less confusing, although it moves the complication to the Terraform code.

Reply:
I would be ok with that if we wanted to go that route. Something like a `bool` attribute called `self_admin` or something along those lines? What do you think @dav3r?

Reply:
Sure, that suggestion makes sense to me- I say go for it.

Reply:
Assuming it works out, we should make the same changes to [cisagov/cool-users-non-admin](https://github.com/cisagov/cool-users-non-admin).